### ISSUE TITLE
Set minimum version in use_package()

### ISSUE THREAD

**Issue by strboul** [r-lib/usethis#386]
I mostly require to have package version next to the package name in the `DESCRIPTION` file.  An additional argument such as `add.version` to the function would be quite useful. For example:

```r
usethis::use_package(package = "dplyr", type = "Imports", add.version = TRUE)
```
Then, it will be adding 'dplyr' to Imports field in DESCRIPTION:
```
Imports: 
    dplyr (>= 0.7.5)
```

**Comment by hadley** [comment]
I suggest we call this `min_version` and accept either `TRUE` (to use the currently installed version) or a string.

**Comment by hadley** [comment]
Related to #498

**Comment by hadley** [comment]
Was fixed in #512
